src/ansys/fluent/core/__main__.py

```diff
@@ -0,0 +1,33 @@
+import os
+import sys
+
+from ansys.fluent.core.launcher.launcher import launch_fluent
+from ansys.fluent.core.session_solver import Solver
+
+
+def setup_for_fluent(version: str, mode: str):
```

Comment:
If we want to move the logic within ansys.fluent.core, we can write a function `get_globals_for_fluent(version: str, mode: str)` which returns a dictionary and inside `setup_for_fluent` we can update the global dict: `globals().update(get_globals_for_fluent(version, mode)`.